.github/workflows/preview.yml

```diff
@@ -94,8 +94,8 @@ jobs:
       - name: Create preview comment - Deploying
         uses: marocchino/sticky-pull-request-comment@v2.8.0
         with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          number: ${{ github.event.pull_request.number }}
+          GITHUB_TOKEN: ${{ secrets.CLERK_COOKIE_PAT }}
+          number: ${{ github.event.issue.number }}
```

Comment:
We want "Clerk Cookie" to comment, so we switch out the token.
The event is coming from a comment, not a label added to a pull request so that needed change, too.

I overlooked those two details when I copied that logic from the clerk-docs workflow